dynacred/src/config.ts

```diff
@@ -0,0 +1,61 @@
+import * as toml from 'toml';
+
+/**
+ * Readonly container for the configuration
+ *
+ * TODO preciser name
```

Comment:
Suggestions?

Reply:
Not really for now, depending on how much data we want to retrieve from the server: For now, the most precise name I can think of is `Conodes`.

Reply:
What is wrong with `Config`? Looks good to me...

Reply:
Now that I'm adding more elements to it, `Config` looks good enough.